sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.2 (2021-06-08)
 - Added method `ChatThreadAsyncClient.listParticipants(ListParticipantsOptions listParticipantsOptions)`
```

Comment:
Are these new public APIs? this will need APIView approval and will also not go out as a patch.. it will have to be a minor release

Reply:
Unreleased Chat